frontend/src/components/StructuredDataScript.tsx

```
@@ -0,0 +1,20 @@
+import { ProfilePageStructuredData } from 'types/profilePageStructuredData'
+
+interface StructuredDataScriptProps {
+  data: ProfilePageStructuredData
+}
+
+// dangerouslySetInnerHTML injects the JSON data as a script tag.
+const StructuredDataScript: React.FC<StructuredDataScriptProps> = ({ data }) => {
+  return (
+    <script
+      type="application/ld+json"
+      dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)